source/source-connector/usage-examples/topic-naming.txt

```diff
@@ -76,13 +81,44 @@ setting as shown in the following example:
 Once set, your connector publishes any changes to the ``data`` collection
 in the ``test`` database to the Kafka topic named ``test.data.mySuffix``.
 
-.. _topic-naming-namespace-map-example:
+.. _topic-naming-namespace-map:
 
 Topic Namespace Map
 -------------------
 
 You can configure your source connector to map namespace values to Kafka
-topic names for incoming change event data.
+topic names for incoming change event data. Topic namespace maps contain
+**pairs** that are made up of a namespace pattern and a destination
+topic.
```

Comment:
> and a destination topic

That's not a topic, but a topic name template. The topic name is then computed based on the namespace that matched the namespace pattern and the corresponding topic name template (note how the text proposed in DOCSP-31502 uses the terms "topic name" and "topic name template").

There are other places where the documentation does not differentiate these two terms, for example:

- _"The following example shows how to specify the `topic.namespace.map` setting to define a topic namespace mappings from the `carDb` database to the `automobiles` topic and the `carDb.ev` namespace to the `electricVehicles` topic: `topic.namespace.map={"carDb": "automobiles", "carDb.ev": "electricVehicles"}`"_. Here `automobiles` is a topic name template, not a topic name. This becomes clear when we read the example further and it says _"If the change event came from the database `carDb` and a collection other than `ev`, the connector sets the destination to the `automobiles.<collection name>` topic."_: the computed topic name is not `automobiles`, but `automobiles.<collection name>`, because `automobiles` is a topic name template. `electricVehicles` obviously must be conceptually the same thing as `automobiles`, i.e., a topic name template. It just so happens that the `electricVehicles` topic name template does not have anything to expand/add (barring the `topic.prefix`/`topic.suffix`), and matches the topic name computed based on it.
- https://github.com/mongodb/docs-kafka-connector/pull/141#discussion_r1346394167

Reply:
Thanks for this explanation, I definitely see your point and understand. Let me go through the page and correct the terminology where needed.